### ISSUE TITLE
Normalize by country population

### ISSUE THREAD

**Issue by wuschi** [lachlanjc/covid19#5]
Hello,
it would be cool if we could normalize the figures by e.g. by infections per million inhabitants.
Italy has ~60 million inhabitants. Comparing against the U.S. (~300 million inhabitants) by total figures doesn't make much sense imho.

**Comment by lachlanjc** [comment]
Yep, totally agree, an option for viewing the lines adjusted per capita would be a great improvement. We’re happily accepting PRs!
